### PR TITLE
fix: solar arc motion, heliacal search limits and CLI JSON sequences

### PR DESCRIPTION
**Pull request by g-battaglia**
Fix three regressions found while reviewing the v6 alpha:

- Solar-arc-directed points now share the progressed Sun's motion, converted to degrees per real-time day. Retrograde flags follow that speed and natal motion-state classifications are cleared. The 1990-01-01 Rome natal directed to 2026 now labels directed Mercury's conjunction with natal Moon as Applying.
- Heliacal event searches stop immediately after collecting the requested count. A single Jupiter rising after 2648-12-01 now returns 2649-02-08 without attempting an out-of-coverage successor search.
- CLI sequence parameters accept JSON arrays and validate structured elements, including `active_aspects=[{"name":"trine","orb":5}]`. Scalar sequences retain comma-separated input support, including literal values starting with `[` that are not syntactically valid JSON.

Includes regression tests and updates to the library/CLI reference documentation.

Validation:

- `uv run poe test:core`: 7,625 passed, 134 skipped, 2 xfailed.
- Targeted predictive, heliacal and CLI tests: 461 passed, 7 skipped.
- Latest CLI suite, including bracket-prefixed CSV regressions: 224 passed.
- Reproduced the Jupiter ephemeris-boundary case against the installed medium kernel.
- Repository-wide Ruff check; mypy and Pyright on the three changed source files; documentation coverage audit; `git diff --check`: passed.

## Summary by CodeRabbit

- **Bug Fixes**
  - Improved CLI sequence parameter handling: valid JSON lists are parsed as structured values, while bracket-prefixed CSV values remain supported.
  - Heliacal event searches now stop after returning the requested number of events.
  - Solar-arc directed charts now report progressed motion speed and retrograde status consistently, with symbolic motion states cleared.

- **Documentation**
  - Clarified CLI sequence parsing, heliacal search limits, and solar-arc motion behavior in the reference documentation.

### CODE REVIEW

**Comment by g-battaglia** [comment]
@coderabbitai review

File: cli/kerykeion_cli/introspect.py [hunk 1]
```diff
@@ -163,6 +163,14 @@ def coerce_value(annotation: Any, raw: str) -> Any:
         return annotation(raw)
     if annotation in (datetime, date):
         return annotation.fromisoformat(raw)
+    if origin in (list, set, frozenset, tuple) or origin in _SEQUENCE_ORIGINS:
+        inner_types = [a for a in get_args(annotation) if a is not ...] or [str]
+        if raw.lstrip().startswith("[") or not all(_is_csv_scalar(t) for t in inner_types):
```
[the remark this answers is not in the document]
Comment (g-battaglia) [reply cli/kerykeion_cli/introspect.py]: Fixed in 8827b90b. Scalar sequences now select JSON only after a syntax check; bracket-prefixed CSV values keep their previous parsing. Structured sequences still require JSON, and valid JSON with invalid element types still raises. Added regressions for lists, abstract sequences and tuples. Validation: 224 CLI tests passed; core suite 7,625 passed, 134 skipped and 2 xfailed; Ruff, mypy and Pyright passed.

**Comment by g-battaglia** [comment]
@coderabbitai review
